webapp/sources/rudder/rudder-web/src/main/resources/demo-rudder-users.xml

```diff
@@ -75,7 +75,6 @@
 
 <authentication hash="bcrypt">
 
-  <!-- Default credentials: admin / admin -->
-  <user name="admin" password="$2b$12$Sz9Arn8QL59mGzt8ykXGhu/onddku53PnjgF6.hmyXnXqHBjUtUJu" role="administrator" />
+  <user name="admin" password="" role="administrator" />
```

Comment:
actually we should keep this until random passwords are actually generated at install